ntoskrnl/ke/amd64/cpu.c

```diff
@@ -35,36 +35,68 @@ static const CHAR CmpIntelID[]       = "GenuineIntel";
 static const CHAR CmpAmdID[]         = "AuthenticAMD";
 static const CHAR CmpCentaurID[]     = "CentaurHauls";
 
+
 /* FUNCTIONS *****************************************************************/
 
 VOID
 NTAPI
 KiSetProcessorType(VOID)
 {
     CPU_INFO CpuInfo;
-    ULONG Stepping, Type;
+
+    struct
+    {
+        union
+        {
+            ULONG Eax;
+            struct
+            {
+                UCHAR Stepping : 4;
+                UCHAR Model : 4;
+                UCHAR Family : 4;
+                UCHAR Unused : 4;
+                UCHAR ModelExt : 4;
+                UCHAR FamilyExt;
+            };
+        };
+    } CPUINFO_EAX;
```

Comment:
Yes, now that's it :slightly_smiling_face: I think I'm going to help with rearranging commits here, and then it will be good to go.